### DISCUSSION TITLE
nCount and nFeature

### DISCUSSION THREAD

**Discussion by DHelix** [stuart-lab/signac#1492] [Q&A] [answered]
Hi, I'm wondering if anyone could explain what the nCount and nFeature in the metadata represent.
Thanks!

**Comment by timoast** [comment, marked as answer]
nCount is the number of total counts for that cell
nFeature is the number of features with >0 counts in the cell

**Reply by DHelix** [reply]
Hi @timoast, 
Thank you for the prompt reply! I generated the Seurat object for the 10X scATAC-Seq data. Does nCount represent the total number of unique fragments in each cell, and nFeature represent the number of peaks detected in each cell? I noticed a high correlation between nCount and nFeature, with nCount being approximately twice the nFeature. Why is that?
Thanks again for your help!

**Reply by timoast** [reply]
please see answer above

**Reply by DHelix** [reply]
Hi @timoast,
Yes, I have read your answer above but I am still not clear what the nCount and nFeature exactly are.
Does the "total counts" mean "total fragments"?
And what exactly is a feature? Does it mean "a peak"?
And why are nCount and nFeature highly correlated?
BTW, I'm working with scATAC-Seq data, generated using 10X Genomics scATAC platform.
Thanks for your time!

**Reply by DHelix** [reply]
I used CountFragments() to calculate the total fragments per cell and the numbers are different from nCounts (their correlation is ~ 0.7). Now I don't think the nCount represents the number of fragments in each cell. 
The nCounts values are generally smaller than the numbers of total fragments, so it doesn't sound like reads or Tn5 cutsites either. 
So what is the "total counts" in your answer? Thanks!

**Reply by timoast** [reply]
Total counts is the total counts for the cell in the count matrix that you provide (the column sums of the matrix). Total fragments is the total number of sequenced DNA fragments for the cell, which could include fragments that are not quantified in the count matrix (falling outside of peaks)

**Reply by DHelix** [reply]
Hi @timoast, I see. Thank you so much for taking the time to answer my questions!
